src/app/src/containers/Profile.jsx

```diff
@@ -1,9 +1,13 @@
+/* eslint-disable */
+/* This component's gradually going to be deprecated in favor of
+   UserProfile.jsx. We retain the file until it has been fully
+   replaced */
+
```

Comment:
Below this there is an `import '../styles/css/filestack.css';` line that we can drop along with removing the actual referenced css file.